azure-pipelines.yml

```diff
@@ -11,12 +11,12 @@ variables:
 pool: 'Self Host Ubuntu'
 
 jobs:
-  - job: Build
+  - job: Validation
```

Comment:
There might be a better name for this--just "Validation" makes it sound like it's running tests, which it isn't.

Reply:
I got rid of this extra job. Putting back inside Build, Test